src/js/reducers/index.js

```diff
@@ -4,7 +4,7 @@ import { NavigationReducer } from '@exponent/ex-navigation';
 // import auth from './auth';
 import profile from './profile';
 import calendar from './calendar';
-import event from './event';
+import event from './event/index';
```

Comment:
@heron2014 At the moment we do.  There's both a file called `event.js` and a folder called `event/` in the same directory.  Eventually `event.js` won't exist anymore, but for now I need both.  And if I don't put `./event/index` then the wrong file is imported